devbox.json

```diff
@@ -1,42 +1,46 @@
 {
-  "packages": [
-    "path:local-flake#bosh-bootloader",
-    "path:local-flake#app-autoscaler-cli-plugin",
-    "path:local-flake#log-cache-cli-plugin",
-    "path:local-flake#uaac",
-    "credhub-cli@latest",
-    "delve@latest",
-    "gh@latest",
-    "go-tools@latest",
-    "gopls@latest",
-    "nodejs@latest",
-    "nodePackages.yaml-language-server@latest",
-    "rubocop@latest",
-    "rubyPackages.solargraph@latest",
-    "swagger-cli@latest",
-    "actionlint@1.6.27",
-    "direnv@2.34.0",
-    "gnumake@4.4",
-    "maven@3.8.6",
-    "pre-commit@latest",
-    "bosh-cli@7.5.6",
-    "golangci-lint@1.57.2",
-    "yq-go@4.43.1",
-    "which@latest",
-    "jq@latest",
-    "fly@7.10.0",
-    "python@latest",
-    "cloudfoundry-cli@8.7.10",
-    "shellcheck@0.10.0",
-    "act@0.2.62",
-    "go@1.21.5",
-    "ruby@3.3.1",
-    "ginkgo@2.17.3",
-    "google-cloud-sdk@latest",
-    "temurin-bin-17@latest",
-    "bundix@latest",
-    "oha@latest"
-  ],
+  "packages": {
+    "path:local-flake#bosh-bootloader":           "",
+    "path:local-flake#app-autoscaler-cli-plugin": "",
+    "path:local-flake#log-cache-cli-plugin":      "",
+    "path:local-flake#mbt":                       "",
+    "path:local-flake#sap-piper": {
+      "platforms": ["x86_64-linux"]
```

Comment:
We build piper for all platforms, why do we only install it on `x86_64-linux`?

Reply:
removed in 2f14adc18abf50a2779a7d2f70c5f8844660fd9f